common/types.go

```diff
@@ -16,35 +16,26 @@
 
 package common
 
-import (
-	"bytes"
-	"fmt"
-)
-
 // Address represents a base58 encoded public key
 type Address string
 
-// Hash used to store a blake2b hash
-type Hash [32]byte
-
-// NewHash casts a byte array to a Hash
-// if the input is longer than 32 bytes, it takes the first 32 bytes
-func NewHash(in []byte) (res Hash) {
-	res = [32]byte{}
-	copy(res[:], in)
-	return res
-}
-
-// ToBytes turns a hash to a byte array
-func (h Hash) ToBytes() []byte {
-	b := [32]byte(h)
-	return b[:]
+// Health is network information about host needed for the rpc server
+type Health struct {
```

Comment:
wondering why this is moved here? is there an import cycle or something?

Reply:
correct, there was an import cycle with network and state